app/controllers/global_search_controller.rb

```diff
@@ -14,9 +12,13 @@ def self.searcher_classes
   end
 
   def index
-    @searchers = self.class.searcher_classes.map do |searcher_class|
-      searcher_class.new(current_user, current_facility, params[:search])
-    end
+    @searchers = if user_signed_in? && !acting_as?
```

Comment:
I am somewhat of the opinion that you should be able to do a little more than you currently can while acting as another user. If we included other searchers, like the order detail or statement searchers, would that lead to some 403s or "you can't do that while acting as another user" from the resulting links? If it does, then this is fine.

If it doesn't I would vote towards including the searchers if you're signed in and using the user you're acting as as the permission check (I forget who is acting_user and who is current_user in this context).

Reply:
You'll get a `This function is unavailable while you are acting as another user.` message if you try to click on an order detail link after searching for it.